Lib/vanilla/vanillaBox.py

```diff
@@ -99,6 +99,14 @@ def __init__(self, posSize):
         self._nsObject.setBoxType_(NSBoxSeparator)
         self._nsObject.setTitlePosition_(NSNoTitle)
 
+    def _getContentView(self):
+        return self._nsObject
+
+    def _breakCycles(self):
+        view = self._getContentView()
+        if view is not None:
+            _breakCycles(view)
+
```

Comment:
there is already a `_breakCycles` in the Box class, this is causing a RecursionError

Reply:
Yeah, I see it. The root of the problem is that `NSBox` with a type `NSBoxSeparator`, which is what we do in lines, doesn't have a `contentView`. When `_getContentView` is called by vanilla's auto layout system, it gets `None` and everything breaks. I overrode `_getContentView` in `_Line` and that caused a problem with `_breakCycles`. I'm working on it.

Reply:
This shuld be fixed with d10098e.